app/assets/javascripts/angular/controllers/flats.js.erb

```diff
@@ -13,7 +14,7 @@ app.controller('ListRoomCtrl', ['$scope', '$http', '$auth', '$location', functio
     Name: 'House'
     }, {
     Id: 3,
```

Comment:
you guys never fixed the indentation here?

![](https://media.giphy.com/media/B0dTbkqIhuiLS/giphy.gif)